### DISCUSSION TITLE
Auto increment starting number

### DISCUSSION THREAD

**Discussion by rynskiy** [directus/directus#15323] [Archive]
Hi,

is it possible to change auto increment starting number by "id" field? We need to have auto increment id field started with number 1000000001. 

If I choose by collection creating Primal Key "id" field type "Auto-incremented integer" > id's will be: 1, 2, 3... But I need to have id's in form of: 1000000001, 1000000002, 1000000003...

Or maybe is it possible to create some own field (not default id) which will be Auto-incremented with custom starting number?

Best regards,
Tony

**Comment by rijkvanzanten** [comment]
Directus actually doesn't control the starting number there. That's a database native thing. You should be able to (re)set the sequence number in most databases though, for example in Postgres, you would use 

```sql
SELECT SETVAL('public.untitled_table_256_id_seq', 1000000001);
```
